src/mcu.cpp

```diff
@@ -1386,6 +1394,7 @@ int main(int argc, char *argv[])
                 printf("  -p:<port_number>               Set MIDI port.\n");
                 printf("  -a:<device_number>             Set Audio Device index.\n");
                 printf("  -ab:<page_size>:[page_count]   Set Audio Buffer size.\n");
+                printf("  -d:<rom_path>                  Set ROM directory.\n");
                 printf("\n");
                 printf("  -mk2                           Use SC-55mk2 ROM set.\n");
```

Comment:
I would maybe separate `-d:` from the others since directories aren't related to audio. Something like this:
```
...
-a:<device_number>             Set Audio Device index.
-ab:<page_size>:[page_count]   Set Audio Buffer size.

-d:<rom_path>                  Set ROM directory.

-mk2                           Use SC-55mk2 ROM set.
-st                            Use SC-55st ROM set.
...
```